src/Evaluate/Infrastructure/Task/Analyze/AnalyzeTask.php

```diff
@@ -76,6 +85,7 @@ public function getCommands(): array
         return [
             new AnalyzeCommand(
                 $this->codeComplianceService,
+                $this->configurationParser,
```

Comment:
As to me it should be on the Codebase level but not inside the command.

Reply:
Moved to Codebase level